api/routes/authRouter/index.js

```diff
@@ -0,0 +1,71 @@
+const express = require("express");
+const DB = require("../../../data/helpers/auth");
+const router = express.Router();
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+require("dotenv").config();
+
+// function helpers
+function generateToken(user) {
+  user = Object.assign({}, user, { password: "******" });
+  const payload = { ...user };
+  const secret = process.env.SECRET;
+  const options = {
+    expiresIn: "48hr"
+  };
+
+  return jwt.sign(payload, secret, options);
+}
+// MIDDLEWARE
+
+// END MIDDLEWARE
+
+router.post("/register", (req, res) => {
+  const user = req.body;
+
+  if (user.name && user.password && user.email) {
+    const pass = bcrypt.hashSync(user.password, 12);
+    user.password = pass;
+    DB.registerUser(user)
+      .then(id => {
+        if (id) {
+          res.status(201).json({ id });
+        } else {
+          res.status(400).json({ error: "User not created, try again." });
+        }
+      })
+      .catch(err => {
+        res.status(422).json({
+          error: "Please check your connection and register again."
+        });
+      });
+  }
+});
+
+router.post("/login", (req, res) => {
+  const { user } = req.body;
```

Comment:
If you did this 
```suggestion
  const { email, password } = req.body;
```

Then you don't need to make two more variables down below